rotkehlchen/tests/api/test_yearn_vaults.py

```diff
@@ -576,11 +576,11 @@ def test_query_yearn_vault_history(rotkehlchen_api_server, ethereum_accounts):
     ))
     assert_simple_ok_response(response)
     with rotki.data.db.conn.read_ctx() as cursor:
-        events = rotki.data.db.get_yearn_vaults_events(
-            cursor,
-            TEST_ACC1,
-            test_vault,
-            rotki.data.msg_aggregator,
+        events = get_yearn_vaults_events(
```

Comment:
why does this appear here? Didn't Isaac fix this test before? Why did he say it passed?

Reply:
Ahhhhhhhh sorry, it's totally my bad. I accidentally amended Isaac's commit